problem_2/utils/finance/finance_manager.py

```diff
@@ -67,4 +67,25 @@ def monthly_returns(df: pd.DataFrame) -> pd.DataFrame:
             # Append the results to the all_monthly_returns DataFrame
             all_monthly_returns = pd.concat([all_monthly_returns, merged_returns], ignore_index=True)
 
-        return all_monthly_returns
\ No newline at end of file
+        return all_monthly_returns
+
+    @staticmethod
+    def covariance_matrix(df: pd.DataFrame, column: str) -> pd.DataFrame:
+        """
+        Calculates the covariance matrix for 'Returns_val' grouped by the 'Ticker'.
```

Comment:
The docstring should be inlined:
""" START HERE,
DO MORE HERE...
END HERE.
"""

according to the PEP8 standard: https://peps.python.org/pep-0008/#documentation-strings

Reply:
Also, I think we should add that the function can raise valueerror in the docstring aswell